lib/reporters/console.js

```diff
@@ -354,7 +354,7 @@ internals.Reporter.prototype.end = function (notebook) {
                 }
             });
 
-            if (coverage.percent < this.settings.threshold || isNaN(coverage.percent)) {
```

Comment:
NaN cannot make it into this branch because of the conditional check above `coverage.percent < 100`